OpenRA.Mods.TS/SpriteLoaders/TmpTSLoader.cs

```diff
@@ -67,8 +85,16 @@ public TmpTSFrame(Stream s, Size size, int u, int v)
 						width += (j < size.Height / 2 - 1 ? 1 : -1) * 4;
 					}
 
-					// TODO: Load Z-data once the renderer can handle it
-					s.Position += size.Width * size.Height / 2;
+					// Unpack depth data
```

Comment:
The following code is identical to the unpacking of the tile data just above. How about extracting it into a method?